web/dashboards/garage/js/garage.js

```diff
@@ -1,327 +1,331 @@
-  getJson(baseUrl + MINUTE_OF_HOUR_ROUTE, function(response) {
-    if (response) {
-      let minuteofhour = response;
-      const trace1 = {
-        x: minuteofhour.x,
-        y: minuteofhour.y,
-        type: "scatter",
-        market: {
-          opacity: 0.7,
-          color: "rgb(49,130,189)"
-        }
-      };
-      const data = [trace1];
-      const layout = {
-        xaxis: {
-          tickangle: -45
-        }
-      };
-      Plotly.newPlot("linechart", data, layout, { showSendToCloud: true });
-
-      // Zone by Time graph
-      getJson(baseUrl + ZONE_BY_TIME_ROUTE, function(response) {
-        if (response) {
-          let zonebytime = response;
-          var data = [
-            {
-              z: zonebytime.z,
-              x: zonebytime.x,
-              y: zonebytime.y,
-              type: "heatmap"
-            }
-          ];
-          var layout = {
-            xaxis: {
-              tickangle: -45
-            }
-          };
-          Plotly.newPlot("heatmap", data, layout, { showSendToCloud: true});
-         }
-      });
-    }
-  });
-}
-
-// GET the JSON response from the given URL
-function getJson(url, callback) {
-  let httpRequest = new XMLHttpRequest();
-
-  httpRequest.onreadystatechange = function() {
-    if (httpRequest.readyState === XMLHttpRequest.DONE) {
-      if (httpRequest.status === 200) {
-        return callback(JSON.parse(httpRequest.responseText));
-      } else {
-        console.log("Error", httpRequest.status, "on GET", url);
-        return callback(null);
-      }
-    }
-  };
-  httpRequest.open("GET", url);
-  httpRequest.setRequestHeader("Accept", "application/json");
-  httpRequest.send();
-}
-
-// Updare Time
-function updateClock() {
-  let now = new Date();
-  dateNode.innerText = now.toLocaleTimeString("en-US",{ hour12:false });
-}
-
-// Parse the config and start periodic data update 
-function handleConfigAndStart(response) {
-  if(response) {
-    config = response;
-    initialiseBeaver(config.hlcServerUrl);
-    updateGraphs();
-    setInterval(updateGraphs, config.updateGraphsMilliseconds);
-  } 
-  else {
-    console.log("Unable to get config.  Refresh page to try again.");
-  }
-}
-
-// The following code runs on startup...
-getJson(baseUrl + CONFIG_ROUTE, handleConfigAndStart);
-setInterval(updateClock, 1000);
+/**
+ * Copyright reelyActive 2019
+ * We believe in an open Internet of Things
+ */
+
+// Constants
+const BASE_ROUTE = "/api/garage";
+const CONFIG_ROUTE = BASE_ROUTE + "/config";
+
+// DOM elementsa
+let temperature = document.querySelector("#temperature")
+let humidity = document.querySelector("#humidity");
+let occupancyCount = document.getElementById("occupancycount");
+let dateNode = document.getElementById("date");
+let presenceLab = [];
+let presenceArray = [];
+let presenceOffice = [];
+let presenceReception = [];
+let managerList = [ "ac233fa152c2", "ac233fa152bb", "ac233fa152a7", "ac233fa152ae", "ac233fa152cc", "ac233fa152a9"];

```

Comment:
we could allow these lists to change dynamically as per the admin instead of having the static the way they are now